var/spack/repos/builtin/packages/accfft/package.py

```diff
@@ -32,15 +32,15 @@ class Accfft(CMakePackage, CudaPackage):
     def cmake_args(self):
         spec = self.spec
         args = [
-            "-DFFTW_ROOT={0}".format(spec["fftw"].prefix),
+            f"-DFFTW_ROOT={spec['fftw'].prefix}",
```

Comment:
self.define(key, value) instead of strings

Reply:
Done!